tests/typ/visualize/gradient-transform.typ

```diff
@@ -0,0 +1,16 @@
+// Test whether `relative: "parent"` works correctly
+// In the first test, the image should look as if there is a single
+// gradient that is being used for both the page and the rectangles.
+// In the second test, the image should look as if there are multiple
+// gradients, one for each rectangle.
```

Comment:
Same comment as in `gradient-relative.typ`, remove the comment or add a unique one.

Reply:
Copy pasting error, I’ll correct once I’m home ;)